### PR TITLE
Moar Quacks

### PR DESCRIPTION
**Pull request by chrisj1**

### CODE REVIEW

**Comment by benjaminrsherman** [comment]
Please remove the quacks on lines 10-14.  Rubber Duck does not own other ducks.

**Comment by chrisj1** [comment]
Done

File: quacks.json [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+    "Ducks do quack and mine have done it at pretty predictable times for pretty ordinary reasons.  If they are outside the coop and hungry or don't have access to water, they'll quack--but that is pretty easy to prevent.  Sometimes, if they are out of the coop, hear my voice and want to find me, they will quack, which usually happens when I go outside during the day, and sometimes in the summer when I'm inside and the windows are open.  Once I show the ducks where I am, they knock it off. When ducks are \"adolescents\" (around 11 to 16 weeks old) I've noticed that they go through a phase when they are more frequently vocal. At this point it is helpful to give them something extra to do, like play in the sprinkler or a big mud puddle.  It also helps to stagger the ages of the ducks so you don't have a bunch of them going through this phase at the same time. We don't have many hawks or predatory birds in our neighborhood, so this is almost never a cause for quacking here.",
```
Comment (benjaminrsherman) [inline quacks.json]: Please remove this line, and the PR will be good.

**Comment by chrisj1** [comment]
Done

**Pull request merged**